src/DigestEmail.php

```diff
@@ -0,0 +1,175 @@
+<?php
+namespace WCEmailDigest;
+use WCEmailDigest\Subscribers;
+
+class DigestEmail extends \WC_Email {
+	/**
+	 * Instance of WCEmailDigest\Subscribers
+	 * @var WCEmailDigest\Subscribers
+	 */
+	public $subscribers;
+
+	public function __construct() {
+		$this->template_base = plugin_dir_path(__DIR__) . 'templates/';
+
+		parent::__construct();
+
+		$this->schedule = $this->get_option('schedule');
```

Comment:
Are these properties owned by `DigestEmail` or `WC_Email`? If the former, can we declare them outside constructor?